packages/compass-import-export/src/import/import-csv.spec.ts

```diff
@@ -255,9 +255,6 @@ describe('importCSV', function () {
           ignoreEmptyStrings: true,
         });
 
-        const errorLog = await fs.promises.readFile(output.path, 'utf8');
```

Comment:
I'm suspicious of this line since the error we're seeing is ENOENT after import seems to have completed. Although with no stack trace for some reason.